src/App.tsx

```diff
@@ -1,61 +1,131 @@
 import './App.scss';
+import { useState, useEffect } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User, Todo } from './components/type';
+
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [copyTodos, setCopyTodos] = useState([...todosFromServer]);
+  const [selectedUserName, getSelectedUserName] = useState('');
+  const [newTitle, getNewTitle] = useState('');
```

Comment:
```suggestion
  const [selectedUserName, setSelectedUserName] = useState('');
  const [newTitle, setNewTitle] = useState('');
```
Why 'get' instead of 'set'?